Test/com/nicoles_change_maker/test/CoinMapTest.java

```diff
@@ -0,0 +1,39 @@
+package com.nicoles_change_maker.test;
+import com.nicoles_change_maker.CoinMap;
+import org.junit.*;
+import java.util.HashMap;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by bears8yourface on 1/5/16.
+ */
+public class CoinMapTest {
+    CoinMap coinMap = new CoinMap();
+    HashMap dictionary = coinMap.dictionary;
+    private static final double DELTA = .001;
```

Comment:
Is this used? The compiler is probably raising a warning if you check in your IDE.